scripts/capture.py

```diff
@@ -8,6 +8,13 @@
 FILE = '%H:%M:%S.png'
 
 
+def rotate_image(image, angle):
+    (h, w) = image.shape[:2]
+    center = (w / 2, h / 2)
```

Comment:
@Adman won't this return a float in Python 3 which might cause trouble later? I guess we are not playing that game at all...